wordpress-mobile/WordPress-Android
# Add bottom sheet after post

Fixes #14704 

This PR adds an empty bottom sheet that's shown after creating a new post. There is a structure in place with a recycler view and adapter to add items to the bottom sheet. The whole functionality is behind a local feature flag.

To test:
- Turn the "Blogging reminders" flag on in App settings/Test feature configuration
- Create a new post
- Notice an empty bottom sheet appears
- Notice it doesn't disappear on rotation
- Notice you can close it

To test:
- Turn the "Blogging reminders" flag off in App settings/Test feature configuration
- Create a new post
- Notice the bottom sheet doesn't appear

## Regression Notes
1. Potential unintended areas of impact
- Nothing

PR submission checklist:
